docs/fsharp/style-guide/formatting.md

```diff
@@ -270,6 +270,17 @@ match x, y with
 | x, y -> 1
 ```
 
+It is qlso accepted to omit parentheses if the tuple is the return value of a function:
+
+```fsharp
+// OK
+let f model msg =
+    match msg with
+    | 1 -> model + 1, []
+    | _ -> model, [ msg ]
+```
+
```

Comment:
```suggestion
In summary, prefer parenthesized tuple instantiations, but when using tuples for pattern matching or a return value, it is considered fine to avoid parentheses.
```